suggester/src/main/java/org/opengrok/suggest/util/Statistics.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * CDDL HEADER START
+ *
+ * The contents of this file are subject to the terms of the
+ * Common Development and Distribution License (the "License").
+ * You may not use this file except in compliance with the License.
+ *
+ * See LICENSE.txt included in this distribution for the specific
+ * language governing permissions and limitations under the License.
+ *
+ * When distributing Covered Code, include this CDDL HEADER in each
+ * file and include the License file at LICENSE.txt.
+ * If applicable, add the following below this CDDL HEADER, with the
+ * fields enclosed by brackets "[]" replaced with your own identifying
+ * information: Portions Copyright [yyyy] [name of copyright owner]
+ *
+ * CDDL HEADER END
+ */
+
+/*
+ * Copyright (c) 2014, 2018, Oracle and/or its affiliates. All rights reserved.
+ */
+
+package org.opengrok.suggest.util;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class Statistics {
+
+  private final long startTime;
+
+  public Statistics() {
+      startTime = System.currentTimeMillis();
+  }
+
+  public void report(Logger log, String msg) {
```

Comment:
It is not a good practice to pass loggers as arguments.

Reply:
I know however in this case it makes sense - otherwise the log message will have a prefix of the `Statistics` class, not the one of the class using it.